packages/jest-jasmine2/src/index.js

```diff
@@ -263,12 +263,15 @@ function jasmine2(config, environment, moduleLoader, testPath) {
   env.addReporter(reporter);
   moduleLoader.requireModule(testPath);
   env.execute();
-  env.snapshotState.snapshot.save();
+  const operationResult = env.snapshotState.snapshot.save();
 
   return reporter.getResults().then(results => {
     results.snapshotsAdded = env.snapshotState.added;
     results.snapshotsRemoved = env.snapshotState.removed;
     results.snapshotsMatched = env.snapshotState.matched;
+    if (operationResult.deleted) {
```

Comment:
I'd do: results.snapshotsFileRemoved = result.removed; where `removed` is a boolean. I don't understand why you used `1` here?